inc/namespace.php

```diff
@@ -125,88 +132,142 @@ function remove_yoast_submenu_page() {
-	add_filter( 'hm.metatags.context.twitter', __NAMESPACE__ . '\\metadata_img_as_tachyon_twitter' );
-	add_filter( 'hm.metatags.context.opengraph', __NAMESPACE__ . '\\metadata_img_as_tachyon_opengraph' );
+function opengraph_presenters( array $presenters ) : array {
+	require_once __DIR__ . '/opengraph/class-altis-opengraph-author-presenter.php';
+	require_once __DIR__ . '/opengraph/class-altis-opengraph-section-presenter.php';
+	require_once __DIR__ . '/opengraph/class-altis-opengraph-tag-presenter.php';
+
+	$presenters[] = new Opengraph\Altis_Opengraph_Author_Presenter();
+	$presenters[] = new Opengraph\Altis_Opengraph_Section_Presenter();
+	$presenters[] = new Opengraph\Altis_Opengraph_Tag_Presenter();
+
+	return $presenters;
+}
+
+/**
+ * Override SEO Social options from config.
+ *
+ * @param array|bool $options Any options set by pre_option_* filters.
+ *
+ * @return array|bool The filtered option values.
+ */
+function override_yoast_social_options( $options ) {
+	$config = Altis\get_config()['modules']['seo']['metadata'] ?? [];
+
+	// Get Opengraph and Twitter card settings. These default to true if the config has been set but no value given to these options.
+	$options['opengraph'] = $config['opengraph'] ?? true;
+	$options['twitter'] = $config['twitter'] ?? true;
+
+	$options['pinterestverify'] = $config['pinterest-verify'] ?? '';
+	$options['facebook_site'] = $config['social-urls']['facebook'] ?? '';
+	$options['twitter_site'] = $config['social-urls']['twitter'] ?? '';
+	$options['instagram_url'] = $config['social-urls']['instagram'] ?? '';
+	$options['linkedin_url'] = $config['social-urls']['linkedin'] ?? '';
+	$options['google_url'] = $config['social-urls']['google'] ?? '';
+	$options['myspace_url'] = $config['social-urls']['myspace'] ?? '';
+	$options['pinterest_url'] = $config['social-urls']['pinterest'] ?? '';
+	$options['youtube_url'] = $config['social-urls']['youtube'] ?? '';
+	$options['wikipedia_url'] = $config['social-urls']['wikipedia'] ?? '';
+
+	// Set fallback image based on config.
+	$options['og_default_image'] = $config['fallback-image'] ?? '';
+	$options['og_default_image_id'] = $config['fallback-image-id'] ?? '';
+
+	// If the fallback image ID was not configured, but we do have a fallback image, get the ID from the URL.
+	if ( empty( $options['og_default_image_id'] ) && $options['og_default_image'] ) {
+		$options['og_default_image_id'] = get_image_id_from_url( $config['fallback-image'] );
+	}
+
+	// These options are only used as fallbacks from the default Home and Front Page SEO options, and possibly not even then.
+	$options['og_frontpage_title'] = $config['opengraph-fallback']['og-frontpage-title'] ?? '';
+	$options['og_frontpage_desc'] = $config['opengraph-fallback']['og-frontpage-desc'] ?? '';
+	$options['og_frontpage_image'] = $config['opengraph-fallback']['og-frontpage-image'] ?? '';
+	$options['og_frontpage_image_id'] = $config['opengraph-fallback']['og-frontpage-image-id'] ?? '';
```

Comment:
Should these fall back to the default image if not set? Not sure if Yoast does that internally or not

Reply:
Apparently not. 👍